komodor/README.md

```diff
@@ -0,0 +1,44 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and to suggest related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect and provides you with the context you need to troubleshoot efficiently and independently.
+
+### Turning troubleshooting chaos into clarity
+Today's microservices systems are complex, distributed and they are constantly changing. Keeping track of so many moving parts in so many places often seems nearly impossible!
+
+Komodor is the missing piece in your DevOps toolchain and offers one unified platform from which you can gain a deep understanding of all of your system events, changes and their effect.
+
+### Empower on-call teams
+Make the knowledge and expertise that has traditionally been held by only a few, clear and visible to Dev and SRE teams.
+
+### Understand your kubernetes
+Gain the K8s visibility you are lacking. See your deployments on a timeline with the relevant information: what changed, what code was pushed and by whom.
+
+### Track your system end-to-end
+View data from your Git, config, infra, alerting and other tools, in one centralized and easy-to-understand display.
+
+### Uncovering the context
+Troubleshoot your microservices based on the most relevant context, connections and dependencies.
+
+### Increase business velocity
+Optimize the troubleshooting process and free your devs-on-call to focus on their daily strategic work. Innovate faster without increasing operational risk.
+
+### Increase performance and availability
+Detect and resolve issues to prevent escalation. Enable any on-call developer to quickly uncover root causes and get the right teams involved for rapid resolution.
+
+### Save on costs and scale smoothly
+Manage your ever-growing system complexity, change frequency and alert volume without having to grow headcount and without an increasing burnout rate. Boost resolution rates, involve fewer people in the process and release DevOps bottlenecks seamlessly.
+
+## Setup
```

Comment:
Hi @angus-young, could you please add steps for how to set up the integration? Do users have to add their Datadog API key somewhere? Is their App key needed? 

What data will be sent into Datadog, and what data will be pulled out?

Reply:
Hi @angus-young - to add onto Alex's comment, please follow the structure of the other readme files with a `##Setup` section that has steps, and a `## Data Collected` section in the precise format. [Auth0](https://github.com/DataDog/integrations-extras/blob/master/auth0/README.md) is a great example if there is not much data collected, and [EventStore](https://github.com/DataDog/integrations-extras/tree/master/eventstore#data-collected) is a good example if you do have metrics.

Reply:
@apigirl I added a ##Data collected section and now i'm getting another error when submitting, is this another version2 bug as I can't find any guidance in the docs for adding Category::Logging to the manifest.json.